frontend/src/scenes/persons/PersonDashboard.tsx

```diff
@@ -0,0 +1,72 @@
+import { HogQLPropertyFilter, PersonType, PropertyFilterType } from '~/types'
+import { Scene } from 'scenes/sceneTypes'
+import { SceneDashboardChoiceRequired } from 'lib/components/SceneDashboardChoice/SceneDashboardChoiceRequired'
+import { SceneDashboardChoiceModal } from 'lib/components/SceneDashboardChoice/SceneDashboardChoiceModal'
+import { sceneDashboardChoiceModalLogic } from 'lib/components/SceneDashboardChoice/sceneDashboardChoiceModalLogic'
+import { useActions, useValues } from 'kea'
+import { Dashboard } from 'scenes/dashboard/Dashboard'
+import { personDashboardLogic } from 'scenes/persons/personDashboardLogic'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { dashboardLogic } from 'scenes/dashboard/dashboardLogic'
+import { useEffect } from 'react'
+
+export function PersonDashboard({ person }: { person: PersonType }): JSX.Element {
+    const { showSceneDashboardChoiceModal } = useActions(sceneDashboardChoiceModalLogic({ scene: Scene.Person }))
+    const { dashboardLogicProps } = useValues(personDashboardLogic)
+
+    // TODO: needs https://github.com/PostHog/posthog/pull/16653 so we can filter by person ID
+    const { dashboard } = useValues(dashboardLogic(dashboardLogicProps))
+    const { setProperties } = useActions(dashboardLogic(dashboardLogicProps))
+    useEffect(() => {
+        if (dashboard && person) {
+            // `dashboard?.filters.properties` is typed as `any` but it's a list...
+            const current = Array.isArray(dashboard?.filters.properties) ? dashboard?.filters.properties : []
+            const hogQLPersonFilter = `person.properties.email = '${person.properties.email}'`
```

Comment:
Is this how it works? Only via email property? 
I set this up locally and tried for user without that set and the whole things was pretty broken. I think if email is required we should make that super obvious. _or_ allow them to specify the user property they care about.

If we keep this flagged, happy for that to be follow up but it was a jarring experience when I tried it.

Reply:
Yep, this is a "for now while behind a flag" thing. I think Simon and Cam use email already.

I'll move the TODO down a few lines `// TODO: needs https://github.com/PostHog/posthog/pull/16653 so we can filter by person ID`